src/metalnx-web/src/main/java/com/emc/metalnx/controller/TemplateController.java

```diff
@@ -301,7 +301,9 @@ public String modifyTemplate(final Model model, @ModelAttribute final MetadataTe
 				throw new Exception("Cannot modify a non-existent template");
 			}
 			DataGridUser loggedUser = loggedUserUtils.getLoggedDataGridUser();
-			if (!template.getOwner().equalsIgnoreCase(loggedUser.getUsername())) {
+
+			// check if user is the template owner or admin
+			if (!(template.getOwner().equalsIgnoreCase(loggedUser.getUsername()) || loggedUser.isAdmin())) {
```

Comment:
so `loggedUser.isAdmin()` is either rodsadmin OR groupadmin?    those are not the same...

I think there are three cases we want to allow editing...

1) loggedUser is template creator/owner
2) loggedUser is rodsadmin
3) loggedUser is groupadmin of group that the template creator/owner is a part of

Reply:
oh I thought `loggedUser.isAdmin()` would return true if `groupadmin` as well.. thanks, will change that.

Reply:
Right, where does that `isAdmin()` code live?   I couldn't find it in a cursory grep...

Reply:
if `isAdmin()` does return true for groupadmin as well, I would argue that is a separate bug to file and to fix...

Reply:
nope, `isAdmin()` only returns true for rodsadmin, and it's inside the DataGridUser class, which is in the jargon layer. There's also an `isGroupAdmin()` available that checks the user type. I will add some code to check if this user is a groupadmin of that template owner's group.

Reply:
perfect.